ams/ams-server/src/main/java/com/netease/arctic/ams/server/terminal/TerminalManager.java

```diff
@@ -139,6 +139,10 @@ public LogInfo getExecutionLog(String sessionId) {
     return new LogInfo(sessionContext.getStatus().name(), sessionContext.getLogs());
   }
 
+  public synchronized void cleanSession() {
+    sessionMap.clear();
+  }
+
```

Comment:
I don't think sessionMap.clear is a good idea.

The session is a heavy object and maybe hold some external resources. close() method should be call if you don't need the session any more.

but don't call session.close() directly in controller.  because: 
- session.close() is a heavy method and maybe failed.
- session.close() may cost times.
- some session may be in running state and can't closed.
mark the session as deperated and close them in gc thread.